### PR TITLE
Add extended operations for ADF V1 multi-nodes Gateway

### PR DESCRIPTION
**Pull request by nicolaszhang**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/datafactory/resource-manager/Microsoft.DataFactoryV1Extended/2017-08-10-preview/swagger/datafactory.json [hunk 1]
```diff
@@ -0,0 +1,774 @@
+{
```
Comment (jianghaolu) [inline specification/datafactory/resource-manager/Microsoft.DataFactoryV1Extended/2017-08-10-preview/swagger/datafactory.json]: This file should appear one level up. We shouldn't have an extra folder `swagger`.

File: specification/datafactory/resource-manager/Microsoft.DataFactoryV1Extended/2017-08-10-preview/swagger/datafactory.json [hunk 2]
```diff
@@ -0,0 +1,774 @@
...
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/Microsoft.DataFactory/datafactories/{dataFactoryName}/gateways/{gatewayName}": {
+      "get": {
+        "tags": [
+          "gatewaysExtended"
+        ],
+        "operationId": "GatewaysExtended_Get",
+        "x-ms-examples": {
+          "GatewaysExtended_Get": {
+              "$ref": "../examples/GatewaysExtended_Get.json"
```
Comment (jianghaolu) [inline specification/datafactory/resource-manager/Microsoft.DataFactoryV1Extended/2017-08-10-preview/swagger/datafactory.json]: When you move it up, remember to fix these example paths.

**Review by jianghaolu — CHANGES_REQUESTED** [review]
The CI is not running properly without the files in the correct location - please address it and I'll take a further look.

**Comment by veronicagg** [comment]
@nicolaszhang any updates?

**Comment by salameer** [comment]
@nicolaszhang  we will close this PR due to lack of response, plz reachout out if you want it to be re-opened

**Pull request closed without merging**